server-profiles/11.2.2/motd

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
I'd remove non-functional files from the profiles like this one, we don't need this info in our provider repo.

Reply:
removed these